docs/source/matlab/common.rst

```diff
@@ -0,0 +1,83 @@
+General key differences
+-----------------------
+
+Data members
+^^^^^^^^^^^^
++-----------------+----------------------+------------------------------------------------------------------------+
+| MATLAB name     | ``pyttb`` name       | Calling convention                                                     |
++=================+======================+========================================================================+
+| ``size``        | ``shape``            | ``X.shape``                                                            |
++-----------------+----------------------+------------------------------------------------------------------------+
+
+Methods
+^^^^^^^
++-----------------+----------------------+------------------------------------------------------------------------+
+| MATLAB name     | ``pyttb`` name       | Calling convention                                                     |
++=================+======================+========================================================================+
+| ``and``         | ``logical_and``      | ``X.logical_and(Y)``                                                   |
++-----------------+----------------------+------------------------------------------------------------------------+
+| ``disp``        | ``__str__``          | ``X``                                                                  |
```

Comment:
Hmm, for all the names `pyttb names` that include dunder methods I wonder if it makes sense to use their public equivalents. Technically they are all hidden methods that get exposed via some other hook. For example `__str__` and `__repr__` both get called under the hood for `str(X)` and `repr(X)`. For someone totally unaware of python just trying to directly translate from matlab based on this doc I'd really hope they don't end up with a bunch of `__str__`, `__rmul__` calls in their code